src/utils/hooks/useListApplications.tsx

```diff
@@ -57,55 +44,30 @@ const useListApplications = (
       userId: currentUser?.userId as number,
       templateCode: type || '',
     },
-    fetchPolicy: 'network-only',
+    fetchPolicy: 'cache-and-network',
   })
 
-  const [getListCount, { data: countData }] = useGetFilteredApplicationCountLazyQuery({
-    fetchPolicy: 'network-only',
-    onCompleted: () => setIsLoadingCount(false),
-  })
+  const applications = data?.applicationList?.nodes ?? []
 
-  useEffect(() => {
```

Comment:
Removing unnecessary state values also means we can get rid of an entire `useEffect` update.